android/src/com/mapswithme/maps/bookmarks/BookmarkCategoriesFragment.java

```diff
@@ -492,12 +482,15 @@ public void process(@NonNull BookmarkCategoriesFragment frag,
         {
           BookmarkManager.INSTANCE.setCategoryName(category.getId(), newName);
         };
-        EditTextDialogFragment.show(frag.getString(R.string.bookmark_set_name),
-            category.getName(),
-            frag.getString(R.string.rename),
-            frag.getString(R.string.cancel),
-            MAX_CATEGORY_NAME_LENGTH,
-            frag);
+        EditTextDialogFragment dialogFragment =
```

Comment:
Looks like this code in `EditAction` class is unreachable because "Edit" action was removed in commit [8dff98cd2fadad2cdab125524def761e34613c23o](https://github.com/organicmaps/organicmaps/commit/8dff98cd2fadad2cdab125524def761e34613c23#diff-de4e7ac21ff23f6e1e90ae66608bf65261d3805453c47b4b95673b0813648130L10-L13)

Reply:
Wait, how category (set) name is changed/edited then?

Reply:
User press "Settings" and adjust name there

Reply:
@biodranik should I remove `EditAction` class from `BookmarkCategoriesFragment.java` ?

Reply:
Yes please. Any cleanup of unused code is welcomed!

Reply:
Ok. Removed EditAction and "Rename" string